framework/decode/vulkan_replay_consumer_base.h

```diff
@@ -1246,6 +1246,37 @@ class VulkanReplayConsumerBase : public VulkanConsumer
     std::vector<uint8_t> matched_replay_cache_data_;
 };
 
+static VkQueue GetDeviceQueue(const encode::VulkanDeviceTable* device_table,
```

Comment:
I feel that it should have a better place to put this function. Please let me know if you know it.

Reply:
framework/graphics/vulkan_resources_util.cpp ? You may also be able to factor some code in that file if you make this function life in there.

Reply:
+1 (if it is easy). `gfxrecon_graphics` is a separate target so if your helper function has too many dependencies on `gfxrecon_decode` then I can see it getting messy and might be better to leave it where it is now.

Reply:
Thanks. I moved this function to `graphics\vulkan_util.h`.